content/minfraud/api-documentation/_schemas/RequestCreditCard.mdx

```diff
@@ -25,9 +26,12 @@ import requestJson from '../_examples/request';
       'Max Length': 4,
     }}
   >
-    The last digits of the credit card number. `last_digits` should be two
-    digits if the issuer ID number sent is eight digits; `last_digits`
-    should be four digits otherwise.
+    The last digits of the credit card number. If you send six digits for the
+    [`issuer_id_number`](#schema--request--credit-card__issuer_id_number), you
+    should send the last four digits for `last_digits`. If you send eight digits
+    for `issuer_id_number`, you should send the last two digits for
+    `last_digits`. If you don't know how many digits you send for
+    `issuer_id_number`, you should send the last two digits for `last_digits`.
```

Comment:
I wonder if this last sentence is necessary. It seems unlikely they wouldn't know (or couldn't figure out with a simple length calculation).